package.json

```diff
@@ -112,6 +112,7 @@
     "eslint-plugin-prefer-arrow": "^1.2.3",
     "eslint-plugin-react": "^7.27.1",
     "eslint-plugin-react-hooks": "^4.3.0",
+    "eslint-plugin-storybook": "^0.5.5",
```

Comment:
Do we want the ^

Reply:
Yes, during the version update storybook suggested something along the lines of "for the best storybook experience we now highly suggest also installing eslint-plugin-storybook" so I went with that which in turn asked to update the .eslintrc file.

Reply:
<img width="947" alt="Screen Shot 2022-01-07 at 8 24 39 AM" src="https://user-images.githubusercontent.com/34781875/148557628-0c9e4adb-76f4-443f-8ebd-f9fa5076ea33.png">

Reply:
I meant the actual caret ^

Reply:
Oops. Fixed!